zowe/zowe-cli
# Move Imperative and CLI log files into same folder

**What It Does**

Fixes https://github.com/zowe/zowe-cli/issues/2116

TODO:
* [x] ~Update default log location in ZE to match~
  * ZE already stores them in the same folder but a different location: `~/.vscode/extensions/zowe.vscode-extension-for-zowe-X.Y.Z/logs`
* [ ] Update default log location in Zowe docs

**How to Test**

Build the CLI from this PR and verify that logs are generated in the new location:
* `~/.zowe/logs/imperative.log`
* `~/.zowe/logs/zowe.log`

**Review Checklist**
I certify that I have:


**Additional Comments**
